src/serial/remote/mod.rs

```diff
@@ -25,6 +25,22 @@ mod protocol_v3;
 mod protocol_v4;
 pub mod riscv_debug;
 
+pub struct RemoteCommands;
+
+#[allow(unused)]
+impl RemoteCommands
+{
+	/// This command asks the probe if the power is used
+	pub const GET_TARGET_POWER_STATE: &'static str = "!Gp#";
+	pub const JTAG_INIT: &'static str = "!JS#";
+	pub const NRST_GET: &'static str = "!Gz#";
+	pub const NRST_SET: &'static str = "!GZ#";
+	pub const PWR_GET: &'static str = "!Gp#";
+	pub const PWR_SET: &'static str = "!GP#";
+	pub const START: &'static str = "!GA#";
```

Comment:
This one should be dropped or made to agree in encoding with https://github.com/P-Storm/bmputil/blob/66efc79859dedbda73c835a433cc1e673a45ab37/src/serial/bmd_rsp.rs#L26 as this is where that get used.

Reply:
As it functions as an ack to the GDB engine to complete any outstanding commands before entering remote protocol mode, yes, it needs to be present.